File: test/ocamlTests/IntTest/powerTest.ml
test "power(7,3)" (fun () -> expect (Int.power 7 3) |> toEqual Eq.int 343) ; 
test "power(0,3)" (fun () -> expect (Int.power 0 3) |> toEqual Eq.int 0) ; 
test "power(7,0)" (fun () -> expect (Int.power 7 0) |> toEqual Eq.int 1) ; 

File: test/fsharpTests/IntTest/powerTest.fs
testCase "power(7,3)" 
<| fun _ -> 
    let expected = 343
    Expect.equal expected (Int.power 7 3) "error"
testCase "power(0,3)" 
<| fun _ -> 
    let expected = 0
    Expect.equal expected (Int.power 0 3) "error"
testCase "power(7,0)" 
<| fun _ -> 
    let expected = 1
    Expect.equal expected (Int.power 7 0) "error"
